DiskIOMeter.c

```diff
@@ -64,7 +64,8 @@ static void DiskIOMeter_updateValues(Meter* this) {
 
       if (data.totalBytesRead > cached_read_total) {
          diff = data.totalBytesRead - cached_read_total;
-         diff /= 1024; /* Meter_humanUnit() expects unit in kilo */
+         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
+         diff = (1000 * diff) / passedTimeInMs; /* convert to per second */
```

Comment:
Possible (minor) error due to order of operations.

The first division drops 10 bits of precision, then things are shifted (no additional information) with the multiplication by 1000, and another ~1 bit is removed (due to the factor `1000/passedTimeInMs`). It might be worth swapping the two divisions, as in that case the division by `passedTimeInMs` would not introduce any error that would be enlarged by scaling the number in the second step (actually the second step would always scale by a factor <1, thus shrinking the error introduced in the first step.

No need to change; just food for thought/discussion.

Reply:
> actually the second step would always scale by a factor <1

What if passedTimeInMs >= 1000?

> It might be worth swapping the two divisions

Why not just `diff = (1000 * diff) / (ONE_K * passedTimeInMs)`?

Reply:
Given `passedTimeInMs` is always around 500ms or greater, there's no real precision to be lost by the second operation, which scales by 1000 / 1024 (which is <1). The error introduced by the initial division is thus scaled down.

Doing the calculation in one step works too, provided that (`diff` being `uint64_t`) it's unlikely we get a diff of ~16EiB in one step.